src/summary/history/encounter.resource.tsx

```diff
@@ -11,3 +11,17 @@ export function getEncounters(
     }
   );
 }
+export function getEncounterById(encounterId: string) {
+  return openmrsFetch(`/ws/fhir/Encounter?${encounterId}`);
+}
+export function getEncounterByUuid(encounterUuid: string) {
+  return openmrsFetch(`/ws/fhir/Encounter?_id=${encounterUuid}`);
+}
+export function searchEncounterByPatientIdentifierWithMatchingVisit(
+  patientIdentifer: string,
+  Visit: string

```

Comment:
What's `Visit`? Is it `visitUuid`? Also it's convention to name variables with small caps

Reply:
noted